src/modules/sensors/sensors.cpp

```diff
@@ -2070,6 +2070,11 @@ Sensors::task_main()
 	/* start individual sensors */
 	int ret = 0;
 
+/*
```

Comment:
Let me know if you will be working on the above problem.  If so, when you are done I will replace my changes to the driver that use the stub class.